packages/web/tests/components/Contributions/ContributionsBox.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '../../testUtils/testTools';
+import { ContributionsBox } from '../../../src/components/Contributions';
+import { ContributionsList } from '../../../src/pages/app/contributions';
+// Mock repo objects
+const clist1: ContributionsList = {
```

Comment:
Since there's only one item here, there's no need to call it a list:
```suggestion
const mockContribution: Contribution = {
```